commands/networkscommands/commands.go

```diff
@@ -14,27 +14,27 @@ func Get() []cli.Command {
 	return []cli.Command{
 		{
 			Name:        "network",
-			Usage:       "Used for Cloud Networks network operations",
+			Usage:       "Set up new networks to use by servers.",
```

Comment:
Updating this to your description. Wish there was some way to say this isn't the only stop - they have to set up the subnet in a separate command here.